playwright/integrations/authenticated-chat-with-reconnect.spec.ts

```diff
@@ -152,4 +152,186 @@ test.describe('AuthenticatedChat @AuthenticatedChatWithChatReconnect', () => {
     });
+
+    test("ChatSDK.startChat() with a valid reconnect id should reconnect to previous chat", async ({ page }) => {
+        await page.goto(testPage);
+
+        const [_, chatTokenRequest, chatTokenResponse, sessionInitRequest, sessionInitResponse, runtimeContext] = await Promise.all([
+            await page.evaluate(async ({ omnichannelConfig, authUrl }) => {
+                const { OmnichannelChatSDK_1: OmnichannelChatSDK } = window;
+
+                const payload = {
+                    method: "POST"
+                };
+
+                const response = await fetch(authUrl, payload);
+                const authToken = await response.text();
+
+                const chatSDKConfig = {
+                    getAuthToken: () => authToken,
+                    chatReconnect: {
+                        disable: false,
+                    },
+                };
+
+                const chatSDK = new OmnichannelChatSDK.default(omnichannelConfig, chatSDKConfig);
+
+                await chatSDK.initialize();
+
+                await chatSDK.startChat();
+
+                location.reload();
+
+            }, { omnichannelConfig, authUrl }),
+
+            page.waitForRequest(request => {
+                return request.url().includes(OmnichannelEndpoints.LiveChatv2AuthGetChatTokenPath);
+            }),
+            page.waitForResponse(response => {
+                return response.url().includes(OmnichannelEndpoints.LiveChatv2AuthGetChatTokenPath);
+            }),
+            page.waitForRequest(request => {
+                return request.url().includes(OmnichannelEndpoints.LiveChatAuthSessionInitPath);
+            }),
+            page.waitForResponse(response => {
+                return response.url().includes(OmnichannelEndpoints.LiveChatAuthSessionInitPath);
+            }),
+            await page.evaluate(async ({ omnichannelConfig, authUrl }) => {
+                const { OmnichannelChatSDK_1: OmnichannelChatSDK } = window;
+
+                const payload = {
+                    method: "POST"
+                };
+
+                const response = await fetch(authUrl, payload);
+                const authToken = await response.text();
+
+                const chatSDKConfig = {
+                    getAuthToken: () => authToken,
+                    chatReconnect: {
+                        disable: false,
+                    },
+                };
+
+                const chatSDK = new OmnichannelChatSDK.default(omnichannelConfig, chatSDKConfig);
+
+                await chatSDK.initialize();
+
+                const runtimeContext = {
+                    requestId: chatSDK.requestId,
+                    authToken
+                };
+
+                const chatReconnectContext = await chatSDK.getChatReconnectContext();
+
+                const { reconnectId } = chatReconnectContext;
+
+                runtimeContext.reconnectId = { reconnectId };
```

Comment:
We should retrieve `reconnectId` instead of creating an object with `reconnectId`

```
runtimeContext.reconnectId = reconnectId;
```